C_CPP/Data Structures/Queue/Reversingqueue.md

```diff
@@ -0,0 +1,61 @@
+# Reversing the Queue 
+
+**QUEUE** - A linear data structure which stores and retrieves data in the **First In First out(FIFO)** order.	
+*Queue Operations:*
+1. *Enqueue* - To add elements in the queue.
+2. *Dequeue* - To delete elements from the queue.
+3. *Queue Front* - Gives the front element of the queue.
+4. *Empty* - To check whether the queue is empty or not.
+
+To perform the reversing of a queue ,we first stored it in a stack and then re-inserted it in the queue.
```

Comment:
```suggestion
To perform the reversing of a queue ,we first stored it in a stack and then re-inserted it in the queue.
```

* Store the elements
in what order ?

How does this re-insert affect the order
I mean elaborate this one-liner properly..
Ofc its simple but still elaborate as much as you can